.yamato/scripts/build_classlibs.bat

```diff
@@ -0,0 +1,20 @@
+@echo OFF
+
+if not exist "incomingbuilds" mkdir "incomingbuilds"
+
+for %%x IN ("windows", "OSX", "Linux") do (
+    ECHO build.cmd libs -os %%x -c release
+    build.cmd libs -os %%x -c release
+    if NOT %errorlevel% == 0 (
+        echo "build failed"
+        EXIT /B %errorlevel%
+    )
+    if not exist "incomingbuilds/coreclrjit-%%x" mkdir "incomingbuilds/coreclrjit-%%x"
+    ECHO xcopy /s /e /h /y "artifacts/bin/runtime/net7.0-%%x-Release-x64" "incomingbuilds/coreclrjit-%%x"
+    xcopy /s /e /h /y "artifacts/bin/runtime/net7.0-%%x-Release-x64" "incomingbuilds/coreclrjit-%%x"
```

Comment:
You can create "functions" in batch files so we don't have to repeat these commands twice:

If you add this label to the end of the file
```
:EchoAndExecute
ECHO %*
CALL %*
GOTO :EOF
```
Then we can do 
```
CALL :EchoAndExecute xcopy /s /e /h /y "artifacts/bin/runtime/net7.0-%%x-Release-x64" "incomingbuilds/coreclrjit-%%x"
```